### ISSUE TITLE
Can't bind to 'scrollParentSelector' since it isn't a known property of 'ngx-image-zoom'.

### ISSUE THREAD

**Issue by rafialikhan** [wittlock/ngx-image-zoom#58]
New version 0.5.1 - breaks during ng build. We were using "ngx-image-zoom": "^0.4.1", in package.json - which now on the build got updated to version 0.4.2

```ERROR in : Can't bind to 'scrollParentSelector' since it isn't a known property of 'ngx-image-zoom'.
1. If 'ngx-image-zoom' is an Angular component and it has 'scrollParentSelector' input, then verify that it is part of this module.
2. If 'ngx-image-zoom' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("th]="200"
    [lensHeight]="100" [maxZoomRatio]="2" [enableScrollZoom]="true" [magnification]="0.3" [ERROR ->][scrollParentSelector]="scrollParentSelector">
  </ngx-image-zoom>
  <img [src]="image" #originalImag")
```
Any idea how to resolve this?

**Comment by wittlock** [comment]
[Version 0.5.0](https://github.com/wittlock/ngx-image-zoom/blob/master/CHANGELOG.md#050) included some breaking changes to try and comply with some Angular design guidelines. One of which is prefixing the selector with lib. The tag is now `<lib-ngx-image-zoom>`
I believe this error will disappear once you update the tags.

I just published a beta release version 0.6.0 as well, where I've rewritten a big chunk of how I figure out the zooming, part of which removed the need for scrollParentSelector. I'd love it if you could try that one out and see if it still works as you expect it to, without using scrollParentSelector at all. I think it might.

**Comment by wittlock** [comment]
Considering this answered a while ago. Please post a new issue of this problem remains in 1.0.1
